internal/coreinternal/go.mod

```diff
@@ -10,6 +10,7 @@ require (
 	github.com/stretchr/testify v1.7.0
 	go.opentelemetry.io/collector v0.46.1-0.20220307173244-f980c9ef25b1
 	go.opentelemetry.io/collector/model v0.46.1-0.20220307173244-f980c9ef25b1
+	go.uber.org/zap v1.21.0
```

Comment:
Why this (minor) change? (and similar once in attributesprocessor)

Reply:
Fixed.